src/components/Section.js

```diff
@@ -2,12 +2,14 @@ export default class Section {
   constructor({ items, renderer }, container) {
     this._items = items;
     this._renderer = renderer;
-    this._container = container;
+    this._container = document.querySelector(container);
   }
   renderItems() {
-    this._items.forEach(this._renderer);
+    this._items.reverse().forEach((item) => {
+      this._renderer(item);
+    });
   }
   addItem(item) {
-    this._container.prepend(item);
+    this._container.append(item);
```

Comment:
Needs correction: The new cards should be added to the beginning of the container

Reply:
Okay I changed it back